backend/app/controllers/api/v1/base_controller.rb

```diff
@@ -20,7 +20,7 @@ def sparse_fieldset
       end
 
       def included_relationships
-        params[:include]&.split(",")
+        params[:includes]&.split(",")
```

Comment:
I had to rename `include` param, because `rswag` complained when I used `include` key. I guess that it is not issue at this part of project, but let me know if you thing otherwise :)